pymc3/sampling.py

```diff
@@ -631,6 +632,51 @@ def sample(
         return trace
 
 
+def subsample(
```

Comment:
Does this need to be an addition to the core sampling module?  If it's step method-specific, as the docstring says, then this wouldn't be the place for it.

Reply:
Hi @brandonwillard No this does not need to be in the core sampling module, we just placed it there as it is related to `sample()`. We can place it in step_methods/mlda.py if this is a better place for it?

Reply:
@brandonwillard Fixed in [17a4f39 ](https://github.com/alan-turing-institute/pymc3/commit/06e43e9464c9541624c783cf046c09c2e42fbd5d)